From 90caa0c3188b2458c89cbd9fe3e609e76d88cb02 Mon Sep 17 00:00:00 2001
From: Lianwei Wang <a22439@motorola.com>
Date: Fri, 9 Oct 2015 16:50:16 -0700
Subject: [PATCH 362/959] IKSWM-6816 kobject: extend uevent keys

The PowerSupply driver send more keys than defined which cause
uevent send failed. Extend the uevent keys to 64 from 32 to fix
it.

Change-Id: Iaaecc4900f477a0df450542f5379d3a12567ad6f
Signed-off-by: Lianwei Wang <a22439@motorola.com>
Reviewed-on: http://gerrit.mot.com/790415
SME-Granted: SME Approvals Granted
SLTApproved: Slta Waiver <sltawvr@motorola.com>
Tested-by: Jira Key <jirakey@motorola.com>
Reviewed-by: Yi-Wei Zhao <gbjc64@motorola.com>
Submit-Approved: Jira Key <jirakey@motorola.com>
---
 include/linux/kobject.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/kobject.h b/include/linux/kobject.h
index 939b11268c8..e96b2d86823 100644
--- a/include/linux/kobject.h
+++ b/include/linux/kobject.h
@@ -28,7 +28,7 @@
 #include <linux/atomic.h>
 
 #define UEVENT_HELPER_PATH_LEN		256
-#define UEVENT_NUM_ENVP			32	/* number of env pointers */
+#define UEVENT_NUM_ENVP			64	/* number of env pointers */
 #define UEVENT_BUFFER_SIZE		2048	/* buffer for the variables */
 
 /* path to the userspace helper executed on an event */
-- 
2.11.0

